src/content/api/programs.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: Programs
+description: Program model and related endpoints
+nav:
+  path: API
+  order: 17
+---
+import Properties from '../../components/Properties.astro';
+import Property from '../../components/Property.astro';
+import Endpoint from '../../components/Endpoint.astro';
+import Badge from '../../components/Badge.astro';
+
+A Program can be used to provide a versatile foundation for diverse organisational requirements.
+Comprising three distinct types - `loyalty`, `acceptance` and `retail`.
+
+- **Loyalty Programs** enable organisations to set up Promotions and Reward their customers.
+A Consumer can connect with a Loyalty Program to participate in Promotions and earn Rewards.
+
+- **Acceptance Programs** allow organisations to create a new [Asset Type](/api/asset-types) on Centrapay’s platform.
+Acceptance Programs allow Merchants to connect with Asset Issuers to accept their Asset Type in-store.
+
+- **Retail Programs** allow Merchants to connect with POS/Terminal providers to activate Centrapay payments on their devices.
+
+## Program Model
+
+<Properties>
+  <Property name="id" type="string">
+    The unique identifier of the Program.
+  </Property>
+
+  <Property name="name" type="string">
+    The display name of the Program.
+  </Property>
+
+  <Property name="type" type="string">
+    The type of a Program. Current supported value is `loyalty`. 'retail' and 'acceptance' at a later date.
+  </Property>
+
+  <Property name="description" type="string">
+    The displayable description for the Program.
+  </Property>
+
+  <Property name="mediaUploadId" type="string">
+    The id of the [Media Upload](/api/media-uploads/) image of the Program.
+  </Property>
+
+   <Property name="img" type="string">
+    The img URL of the Program.
+  </Property>
+
+  <Property name="accountId" type="string">
+    The Centrapay accountId for an org account.
```

Comment:
Copying from Collections: 'The Account that will own the Program.' (include a link to the Account model)